src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -204,6 +225,34 @@ private ArgumentsBuilder createBaselineArgumentsForFuncotator(final String varia
 
     //==================================================================================================================
 
+    @DataProvider
+    private Object[][] provideForNonTrivialLargeDataTest() {
+
+        return new Object[][] {
+                {
+                        FuncotatorTestConstants.REGRESSION_TEST_HG19_DATA_SET_1,
```

Comment:
Rename constants to match test case name

Reply:
FIXED.